cfe_internal/enterprise/federation/federation.cf

```diff
@@ -88,9 +119,22 @@ bundle agent transport_user
       depth_search => default:recurse_with_base("inf"),
       file_select => default:all,
       perms => default:mog( "600", $(user), "root" );
+    "$(home)/.ssh/authorized_keys"
```

Comment:
$(ssh_auth_keys) here instead of $(home)/.ssh/authorized_keys

Reply:
ack.